felix/bpf/xdp/attach.go

```diff
@@ -73,7 +73,7 @@ func (ap AttachPoint) ProgramName() string {
 	return "xdp_calico_entry"
 }
 
-func (ap *AttachPoint) Log() *log.Entry {
+func (ap AttachPoint) Log() *log.Entry {
```

Comment:
Is it better to remove the '*' or to add it? If we remove it, we keep copying the structure on each call  where we pass it. I am not saying one way or another, just reminding you that it makes a difference.